cmp/grid/Grid.js

```diff
@@ -349,7 +343,7 @@ class LocalModel {
 
             let icon = displaySpec.icon;
             if (isValidElement(icon)) {
```

Comment:
This seems wrong to me.  Think you need to test this.

Reply:
(Probably want to leave this one as is.)

Reply:
It works in toolbox!